Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -47,27 +50,30 @@ then
 	exit 1
 fi
 
-while getopts n:m:w:s: option
+while getopts n:m:w:l:s: option
 do
 	case "${option}"
 	in
 		n) NUM_VEHICLES=${OPTARG};;
 		m) VEHICLE_MODEL=${OPTARG};;
 		w) WORLD=${OPTARG};;
+        l) LABEL=${OPTARG};;
```

Comment:
you can use the `iris_rtps` model. There is no need to add this new option.

Reply:
To use iris_rtps, I think iris_rtps.sdf should be implemented.

Reply:
It is https://github.com/PX4/sitl_gazebo/tree/master/models/iris_rtps :)

Reply:
> you can use the `iris_rtps` model. There is no need to add this new option.

Also, the option is for build directory. For example, "-l default" for px4_sitl_default or "-l rtps" for px4_sitl_rtps.

Reply:
Yes but then you don't need to export the label as an env variable as you are doing bellow. And I will call this instead `t` for (build) target, which is the case.

Reply:
I also agree that specifying the build target explicitly would be the nicer approach

Reply:
OK. I will add "t" option to specify target explicitly.